src/ansys/aedt/core/visualization/post/post_3dlayout.py

```diff
@@ -161,7 +161,7 @@ def _compute_power_loss(self, net_filter=None, layer_filter=None, solution=None)
 
     @pyaedt_function_handler()
     def compute_power_by_layer(self, layers=None, solution=None):
-        """Computes the power by layer. This applies only to SIwave DC Analysis.
+        """Compute the power by layer. This applies only to SIwave DC Analysis.
 
```

Comment:
```suggestion
        """Compute the power by layer.
        
        This applies only to SIwave DC Analysis.

```
I would keep a short title description and add the rest as body content.